scene/gui/control.cpp

```diff
@@ -3506,8 +3502,10 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_layout_direction"), &Control::get_layout_direction);
 	ClassDB::bind_method(D_METHOD("is_layout_rtl"), &Control::is_layout_rtl);
 
+#ifndef DISABLE_DEPRECATED
```

Comment:
Technically the C++ methods should also be wrapped, but if we use `DISABLE_DEPRECATED` only to keep track of things, that would be unnecessary.